torch/backends/cudnn/rnn.py

```diff
@@ -184,7 +184,7 @@ def _copyParams(params_from, params_to):
         # use biases, zip will terminate once layer_params_from ends and ignore them.
         for param_from, param_to in zip(layer_params_from, layer_params_to):
             assert param_from.type() == param_to.type()
-            param_to.copy_(param_from, broadcast=False)
+            param_to[:] = param_from.view_as(param_to)
```

Comment:
any reason why `copy_` became incorrect in here?

Reply:
We don't support `broadcast=False` on Variables.

Reply:
This could just be:

```
param_to.copy_(param_from.view_as(param_to)
```